go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/google/docsy-example
+
+go 1.13
+
+require (
+	github.com/FortAwesome/Font-Awesome v0.0.0-20210804190922-7d3d774145ac // indirect
+	github.com/google/docsy v0.1.1-0.20220321183617-02df04c0f2d4 // indirect
+	github.com/google/docsy/dependencies v0.2.0-pre // indirect
```

Comment:
@deining - should `v0.2.0-pre` here be changed to `0.2.0-pre` (in this file and go.sum)?

Reply:
Yes that makes sense. Both dependencies `docsy` and `docsy/dependencies` now point to the same revision`02df04c0f2d4`.